src/storage.ts

```diff
@@ -0,0 +1,37 @@
+if (typeof localStorage === 'undefined' || localStorage === null) {
```

Comment:
this definitely should be something you can pass in a constructor, etc vs always using same global object. Like e.g. when you are running tests you most likely would want to use in-memory storage. Some web apps would want to use `sessionStorage` instead of `localStorage`, etc.

Reply:
That seems like it would add a bit of complexity to a currently simple interface, when we're not sure anyone cares. Unless I'm mistaken, the tests should pass just fine even if they write some data to a file.

But thanks for the comment! I noticed that I set the localStorage directory to `./scratch` by accident, which is not what I meant to do. 

I've updated it to this instead, which will allow CLI people to control where the cache directory goes while providing a reasonable default:

```ts
if (typeof localStorage === 'undefined' || localStorage === null) {
    const { LocalStorage } = require('node-localstorage'); // eslint-disable-line @typescript-eslint/no-var-requires
    localStorage = new LocalStorage(process.env.NEAR_CACHE_DIR || '~/.near-api-js');
}
```

Marking this as resolved, but feel free to reopen the discussion if you feel strongly that we need to provide more configurability for the initial release of this feature.